test/test_ops.py

```diff
@@ -171,6 +171,27 @@ def unsupported(dtype):
 
         self.assertEqual(supported_backward_dtypes, claimed_backward_supported, msg=msg)
 
+    # Validates that each OpInfo works correctly on different CUDA devices
+    @skipCUDAIfRocm
+    @onlyCUDA
+    @ops(op_db, allowed_dtypes=(torch.float32, torch.long))
+    def test_multiple_devices(self, device, dtype, op):
+        if torch.cuda.device_count() < 2:
```

Comment:
We actually have another decorator for this:

https://github.com/pytorch/pytorch/blob/06fc637b41df1181b32d06cc3c5791965d85395c/torch/testing/_internal/common_device_type.py#L181

When passed the `device` kwarg (more correctly the `devices` kwarg at that point) will be a list of string representations of the devices.

So I think this should just add `@deviceCountAtLeast(2)` and then it can acquire the first and second device representations from the devices list

Reply:
Thanks! I'll use that